t/unit/backends/test_dynamodb.py

```diff
@@ -158,6 +158,55 @@ def test_wait_for_table_status(self):
         self.backend._wait_for_table_status(expected='SOME_STATE')
         assert mock_describe_table.call_count == 2
 
+    def test_update_table_ttl_change_pending(self):
+        self.backend._client = MagicMock()
+        mock_describe_ttl = self.backend._client.describe_time_to_live = MagicMock()
```

Comment:
[moto](https://github.com/spulec/moto) supports table TTL, so you could use it here instead of creating Mock objects, as a means of functional testing, especially since the local DynamoDB does not support TTL yet.